client/blocks/signup-form/index.jsx

```diff
@@ -262,6 +293,16 @@ class SignupForm extends Component {
 				}
 			}
 
+			if ( messages.first_name ) {
```

Comment:
Why use the snake_case notation at all? Seems a hassle to have to convert things.

Reply:
This is the 'raw' data from the API hence the need to convert.  
I cleaned it up a little by using `mapKeys` and `camelCase` when it's loaded.